config/common_versions.yml

```diff
@@ -0,0 +1,5 @@
+title: v3.17.1
```

Comment:
Is title needed here? I can imagine it is. If required I'd prefer it be set to unused or something like that.

Reply:
So literally put in "unused"?